tools/retip/retip_apply.xml

```diff
@@ -0,0 +1,31 @@
+<tool id="retip_apply" name="Retip prediction" version="@VERSION@">
```

Comment:
```suggestion
<tool id="retip_apply" name="Retip prediction" version="@VERSION@+galaxy0">
```

Reply:
Thanks @martenson for this information. I am planning to use @TOOL_VERSION@ instead of @VERSION@ because in the galaxy tools documentation they have mentioned this name (although I have seen some tools on IUC that have used @VERSION@. Probably this @TOOL_VERSION@ change came recently.).

Reply:
The actual name is fine, important is the semantics of the version itself I think.